MobileCenter/MobileCenter/Internals/Channel/MSLogManagerDefault.m

```diff
@@ -83,7 +83,15 @@ - (void)enumerateDelegatesForSelector:(SEL)selector withBlock:(void (^)(id<MSLog
 - (void)channel:(id<MSChannel>)channel willSendLog:(id<MSLog>)log {
   [self enumerateDelegatesForSelector:@selector(willSendLog:)
                             withBlock:^(id<MSLogManagerDelegate> delegate) {
-                              if ([[delegate groupId] isEqualToString:[channel.configuration groupId]]) {
+
+                              /*
+                               * If the delegate doesn't have groupId implementation, it assumes that the delegate is
+                               * interested in all kinds of logs.
+                               */
+                              if (![delegate respondsToSelector:@selector(groupId)] ||
+
+                                  // If the delegate has groupId implementation, compare groupId.
```

Comment:
It's kind of hard to read having the 2 conditions from the same `if` separated by a new line and a comment. Could you merge this comment with the comment just above ?